api/controllers/shabads.js

```diff
@@ -362,6 +363,7 @@ exports.hukamnamas = async (req, res) => {
   }
   if (!q) {
     q = 'SELECT ID as hukamDate, ShabadID FROM Hukamnama ORDER BY ID DESC LIMIT 1';
+    output.is_latest = true;
```

Comment:
should this be camelCase (`isLatest`)?  to be consistent with other object properties we have 
like `shabadIds` below

Reply:
I thought it looked weird.